packages/forms/src/UIForm/fieldsets/Array/Array.component.js

```diff
@@ -137,24 +133,25 @@ export default class ArrayWidget extends React.Component {
 	}
 
 	render() {
-		const { errorMessage, isValid, schema } = this.props;
+		const { displayMode, schema } = this.props;
 		const canReorder = schema.reorder !== false;
 
-		const templateId = 'array';
-		const ArrayTemplate = this.props.templates[templateId] || defaultTemplates[templateId];
+		const baseTemplateId = 'array';
+		const templateId = `${baseTemplateId}_${displayMode}`;
+		let ArrayTemplate = this.props.templates[templateId] || defaultTemplates[templateId];
```

Comment:
i would extract all those ArrayTemplate (why starting with a capital letter ?) inside a function, it is hard to read

Reply:
Because it feels familiar quand you write `<ArrayTemplate>` after, instead of `<arrayTemplate>`